tracing/src/lib.rs

```diff
@@ -237,7 +237,16 @@
 //! ```
 //!
 //! The span macros also take a string literal after the level, to set the name
-//! of the span.
+//! of the span (as above).  In the case of the event macros, the name of the event can
+//! be overridden (the default is `event file:line`) using the `name:` specifier.
+//! 
```

Comment:
looks like rustfmt doesn't like the whitespace here: https://github.com/tokio-rs/tracing/actions/runs/6088114686/job/16519484253?pr=2699
```suggestion
//!
```